certs/certs.c

```diff
@@ -6,7 +6,7 @@
 #include "certs.h"
 
 const char certificates[] =
-/* DigiCert Baltimore Root */
+/* DigiCert Baltimore Root --America--*/
```

Comment:
>America [](start = 29, length = 7)

i don't think this is necessarily true either. this one i believe is essentially used if `!China && !Germany`. More specifically hubs with a tail endpoint of `.azure-devices.net`

Reply:
Yeah, a closer read of that article you linked me seemed to indicate that those other certs are mostly for servers geolocal to those regions that are seperate for compliance etc reasons. So !China and !Germany seems fair.

Reply:
I put Global though to be more succinct.